docs/source/run/parameters.rst

```diff
@@ -1039,3 +1039,19 @@ Whether the energy loss due to classical radiation reaction of beam particles is
     Whether the beam particles undergo energy loss due to classical radiation reaction.
     The implemented radiation reaction model is based on this publication: `M. Tamburini et al., NJP 12, 123005 <https://doi.org/10.1088/1367-2630/12/12/123005>`__
     In normalized units, `hipace.background_density_SI` must be specified.
+
+Spin tracking
+-------------
+
+Track the spin of each beam particle as it is rotated by the electromagnetic fields using the
+TBMT model. This will add three extra components to each beam particle to store the spin and output
```

Comment:
Could be helpful to add a link or ref for `TBMT model`?

Reply:
Agreed, it could be:
> Thomas–Bargmann–Michel–Telegdi (TBMT) model, see [Z. Gong et al., Matter and Radiation at Extremes 8.6 (2023), https://doi.org/10.1063/5.0152382] for the details of the implementation.